pkg/server/restart.go

```diff
@@ -391,7 +391,7 @@ func loadImages(ctx context.Context, cImages []containerd.Image, provider conten
 		// Checking existence of top-level snapshot for each image being recovered.
 		if _, err := snapshotter.Stat(ctx, image.ChainID); err != nil {
 			glog.Warningf("Failed to stat the top-level snapshot for image %+v: %v", image, err)
-			continue
+			// TODO(random-liu): Consider whether we should try unpack here.
```

Comment:
Seem like we do not need it here. Because we will do it during CreateContainer.

Reply:
Usually, we assume container creation should be fast. We may want to try avoiding unpack before container creation as much as possible. :)

Reply:
Yeah but on a restart, it probably does not matter if the delay for unpack is attributed to the create should all add up to the same cost?

Reply:
@mikebrow Yeah, the cost is the same. We just need to decide whether we want a faster restart or faster container creation.

Reply:
vote for faster container creation..

Reply:
I feel like fast restart may be more important. :p Slow container creation only affect one container, but slow restart affect the whole node.

Reply:
hmm can we have both faster restart and faster container creation when not restarting :-)